### PR TITLE
Update User Research page “getting-informed-consent-for-user-research”

### PR DESCRIPTION
**Pull request by lookupdaily**
This is a manually created pull request for content updates for the User research page "Getting informed consent for user research", created by @johnwaterworth using the Netlify CMS

Deploys on this branch are currently failing, due to uninstalled gems during build. As we do not have access to create a clean deploy through the Netlify UI I have created a new branch to create a clean slate for this change.

### CODE REVIEW

**Comment by johnwaterworth** [comment]
Thanks Liz. The researchers have reviewed the updated content in a Google Doc. So here we just need to check the markdown, links, etc.

File: src/user-research/getting-informed-consent-for-user-research.md [hunk 1]
```diff
@@ -3,140 +3,117 @@ title: Getting informed consent for user research
...
+On different projects you might use dxw branded consent materials and follow our processes, or use consent materials provided by the client and follow their processes.
+
+There are three basic options:
 
-### Using dxw or client consent materials
+1. Use dxw consent materials and processes.
 
-There are two basic options:
+   In this case, we use our own [templates]
+(https://docs.google.com/document/d/1ilYXwrrkl9s4jMRkJm3uCWAON6bnEeYinKvPd-VOz8k/) to create dxw branded consent materials.
```
Comment (lookupdaily) [inline src/user-research/getting-informed-consent-for-user-research.md:19]: Just one link issue here, as the link (in brackets) has moved to a new line. This should be resolvable via the markdown editor in GitHub, but let me know if you would like me to fix

**Review by lookupdaily — COMMENTED** [review]
There is just one formatting md which needs resolving, otherwise the links all look good

File: src/user-research/getting-informed-consent-for-user-research.md (hunk 1 above)
Comment (johnwaterworth) [reply src/user-research/getting-informed-consent-for-user-research.md]: Thanks. Have fixed that one.

**Review by johnwaterworth — APPROVED** [review]
Has been reviewed by Liz. And issue fixed by me.

**Pull request merged**